sycl/unittests/pi/piPlatformTest.cpp

```diff
@@ -0,0 +1,104 @@
+//==---- piPlatformTest.hpp --- PI unit tests
+//-------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "CL/sycl/detail/pi.hpp"
+#include <CL/sycl.hpp>
+#include <gtest/gtest.h>
+#include <memory>
+
+using namespace cl::sycl;
+
+namespace {
+class piPlatformTest : public ::testing::Test {
+protected:
+  piPlatformTest() { detail::pi::piInitialize(); }
+
+  ~piPlatformTest() = default;
+};
+
+TEST_F(piPlatformTest, piPlatformGetInfo) {}
```

Comment:
Why do you need this empty function?

Reply:
> // these tests are intentionally empty, defining placeholder test cases for
> // additional patches
Although I'm not sure if we need them. Isn't a todo comment enough?

Reply:
Looks like this particular one is implemented in `piPlatformsGetInfo`

Reply:
I can remove these, I figured they were useful for demonstrating the separation of unit tests by topic (piPlatformTest tests piPlatform* functions etc) but I can understand it would be better to just have the todo.

Reply:
My mistake, I misread the diff and thought this was in reference to the empty tests from the other source file. I have removed this duplicate function.